samplers/parser.go

```diff
@@ -69,12 +70,48 @@ func ConvertMetrics(m *protocol.Message) ([]UDPMetric, error) {
 		}
 		metrics = append(metrics, metric)
 	}
+	indicators, err := metricsFromIndicatorSpan(m)
```

Comment:
Should we make this a configurable action? It seems useful to disable for write-amplification reasons, etc.

Reply:
Yes! I think this goes very well together with your feedback about the name - I'll make the name configurable, and if the name isn't set, we don't report.